nubis/puppet/pocket.pp

```diff
@@ -0,0 +1,63 @@
+cron::daily { "${project_name}-pocket":
+  hour    => '10',
+  minute  => fqdn_rand(60),
+  user    => 'etl',
+  command => "nubis-cron ${project_name}-pocket /opt/etl/pocket/run",
+}
+
+package { 'pyodbc':
```

Comment:
This is already defined in nubis/puppet/adi.pp and will cause a redefinition error.
Now that 2 jobs require pyodbc, I would move the pyodbc package definition into nubis/puppet/packages.pp